includes/Shopping/Shopify_Query.php

```diff
@@ -198,9 +199,13 @@ protected function get_products_query( string $search_term ): string {
 	 * @since 1.21.0
 	 *
 	 * @param string $search_term Search term to filter products by.
+	 * @param string $orderby Sort retrieved products by parameter.
+	 * @param string $order   Whether to order products in ascending or descending order.
+     *                        Accepts 'asc' (ascending) or 'desc' (descending).
```

Comment:
```suggestion
	 *                        Accepts 'asc' (ascending) or 'desc' (descending).
```

My fault for mixing tabs and spaces in my previous suggestion